dialog_flow_chatgpt_skill/dnnc/server.py

```diff
@@ -0,0 +1,71 @@
+import logging
+import time
+import os
+import random
+from argparse import Namespace
+
+import torch
+import sentry_sdk
+from flask import Flask, request, jsonify
+from sentry_sdk.integrations.flask import FlaskIntegration
+
+from models.utils import load_intent_datasets, sample
+from models.dnnc import DNNC
+from intent_predictor import DnncIntentPredictor
+
+sentry_sdk.init(dsn=os.getenv("SENTRY_DSN"), integrations=[FlaskIntegration()])
+
+logging.basicConfig(format="%(asctime)s - %(name)s - %(levelname)s - %(message)s", level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+random.seed(42)
+
+DEFAULT_CONFIDENCE = 0.9
+ZERO_CONFIDENCE = 0.0
+MAX_SEQ_LENGTH = 128
+TRAIN_FILE_PATH = os.getenv("TRAIN_FILE_PATH")
+DEV_FILE_PATH = os.getenv("DEV_FILE_PATH")
+MODEL_PATH = "roberta_nli"
+
+try:
+    train_examples, dev_examples = load_intent_datasets(TRAIN_FILE_PATH, DEV_FILE_PATH, True)
```

Comment:
dev_examples is unused.
```suggestion
    train_examples, _ = load_intent_datasets(TRAIN_FILE_PATH, DEV_FILE_PATH, True)
```